dev/ubuntu20.04.Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:20.04
+WORKDIR /home/
+
+RUN ["apt-get", "-y", "upgrade"]
+RUN ["apt-get", "-y", "update"]
+RUN ["apt-get", "install", "-y", "git"]
+RUN ["git", "clone", "https://github.com/GaloisInc/cryptol.git"]
+
+COPY dev_setup.sh ./cryptol/
```

Comment:
todo: oops, this is a testing artifact. I will update this to run the setup script in the repo instead. Ditto below.

Reply:
Fixed this. It technically won't work for anyone testing before this is merged because the dev setup script isn't in main; you can test locally by adding `"-b", "add-ubuntu-dev-setup"` to the clone line.